erts/emulator/nifs/common/zstd_nif.c

```diff
@@ -820,29 +820,34 @@ static ERL_NIF_TERM create_cdict_nif(
     ErlNifBinary bin;
     int level;
 
-    if (!enif_inspect_iolist_as_binary(env, argv[0], &bin) ||
-        !enif_get_int(env, argv[1], &level)) {
+    if (!enif_is_binary(env, argv[0]) || !enif_get_int(env, argv[1], &level)) {
```

Comment:
Do caller code or docs need update about iolist not being accepted anymore?

Reply:
No, the spec was like that already. The nif implementation just didn't respect it.